internal/main.go

```diff
@@ -94,7 +94,7 @@ func main() {
 	var dataOut []byte
 	if downtranslate {
 		// translate from 3 to 2
-		cfg, rpt, err := v3_0.Parse(dataIn)
+		cfg, rpt, err := v3.Parse(dataIn)
```

Comment:
Ignition 2.x, unlike 0.x, doesn't automatically chain to the previous spec version if it can't parse with the specified one.  So we need to do that ourselves here, or else we won't be able to parse 3.0 specs.

Reply:
Fixed :)